daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -374,6 +397,16 @@ private[lf] object Speedy {
       kontStack.remove(kontStack.size - 1)
     }
 
+    @inline
+    private[speedy] def peekKontStackEnd(): Kont = {
+      kontStack.get(kontStack.size - 1)
+    }
```

Comment:
Why do we need braces ?

Reply:
It is more of a personal preference. I generally like to make things as explicit as possible (even when short cuts may exist). Happy to alter if there's a strong opinion here though.

Reply:
1. In my point of view the mandatory formatting already make it explicit enough.
2. I prefer reserving braces for multi-statements in both named and anonymous function. It is not a strong preference though, so feel free to do as you like.